SPECS/lua/lua.spec

```
@@ -85,7 +69,7 @@ cat > %{buildroot}%{_libdir}/pkgconfig/lua.pc <<- "EOF"
     Libs: -L${libdir} -llua -lm
```

```suggestion
    Libs: -L${libdir} -llua -lm -ldl
```

This PR looks good, although I think we may be missing a flag on this line. I attempted to do this upgrade back in February (#630), but it fell by the wayside. When I was researching this upgrade, I cross-referenced the [LFS page](https://www.linuxfromscratch.org/blfs/view/8.2/general/lua.html) since that seems to be the origin for these build instructions. It looks like we need the `-ldl` flag here to match with the pkg-config file from LFS.

---

Awesome, thank you for letting me know, Thomas!